src/tasks/python/PythonExtensionHelper.ts

```diff
@@ -36,7 +36,7 @@ export namespace PythonExtensionHelper {
 
     export async function getPythonExtension(): Promise<vscode.Extension<PythonExtensionAPI>> | undefined {
         const pyExtensionId = 'ms-python.python';
-        const minPyExtensionVersion = new semver.SemVer('2020.5.78807');
+        const minPyExtensionVersion = new semver.SemVer('2020.5.78807'); // TODO
```

Comment:
Maybe add more of a comment as to what's needed to do?

Reply:
Yeah, I put this in to remember to bump the new min version before submitting. I'm still trying to figure out what the new min version ought to be.